Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -695,9 +695,29 @@ def test_log
         system "git", "commit", "--allow-empty", "-m", "This is a test commit"
       end
     end
+
+    core_tap = CoreTap.new
```

Comment:
@UniqMartin I think it might be nice to keep the `test_$CMDNAME` format for the integration tests. This "write a formula into the core tap` is used in multiple places in this file, though, and could be pulled into a utility method.